README.md

```diff
@@ -45,34 +45,53 @@ default:
   suites:
     default:
       contexts:
-        - IntegratedExperts\BehatRelativity\RelativityContext:
-          -
-            # Supported screen breakpoints.
-            breakpoints:
-              mobile:
-                width: 320
-                height: 480
-              tablet:
-                width: 768
-                height: 1024
-              desktop:
-                width: 992
-                height: 1024
-                default: true
-              desktop_large:
-                width: 1200
-                height: 900
-            # Vertical offset.
-            offset: 60
-            # List of site-wide components.
-            components:
-              'page': "#page"
-              'main': "#main"
-              'top': "#top"
-              'bottom': "#bottom"
+        - IntegratedExperts\Behat\Relativity\Context\RelativityContext
```

Comment:
Please convert the namespacing to the format like in behat-screenshot:
`IntegratedExperts\BehatRelativity\Context\RelativityContext`